### ISSUE TITLE
Launcher config edit button is confusing when disabled

### ISSUE THREAD

**Issue by WeirdRubberDuck** [OpenSpace/OpenSpace#3443]
I find the “disabled” state of buttons really hard to understand. Here, the “Edit” button to me rather looks visually highlighted than disabled.  (see image below)

Why is this “disabling” even needed?  That is, why can’t I open and edit it to make a copy, like I can with the read-only profiles?

~~The best solution to this issue would probably be to make both these views work similarly and avoid this weird highlighting of the disabled button.  (Not disabling buttons seems to be the preferred approach when it comes to accessibility, as well, according to the internet)~~  Not an option. Se comment below from Alex

![image](https://github.com/user-attachments/assets/1ec06bbb-737e-445e-8a33-fa96728999d8)

**Comment by alexanderbock** [comment]
Agree that the color should be different.  The reason why that edit button can be disabled but the Profile one cannot is that we decided when designing the Window Options editor that it shouldn't be capable of editing any arbitrary configuration file.  That means that only configuration files that have been created with the editor are editable as we otherwise cannot guarantee that a simple load->save operation doesn't remove settings that a user put into the json file by hand.

So basically, if a config file doesn't have the a key
```json
  "generator": {
    "major": 1,
    "minor": 1,
    "name": "SgctWindowConfig"
  }
```
in it, the edit button is disabled
